CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## Wednesday, April 24th, 2024
+
+### Added
+
+* support for a local .env file 
```

Comment:
Make this a bit more explicit?  (e.g., "get configurables from the environment, and provide a sample .env file which can be copied for local development").

Reply:
Done!